codegen/templates/tchannel_endpoint.tmpl

```diff
@@ -81,12 +81,12 @@ func (h *{{$handlerName}}) Handle(
 	ctx context.Context,
 	reqHeaders map[string]string,
 	wireValue *wire.Value,
-) (isSuccessful bool, response zanzibar.RWTStruct, headers map[string]string, e error) {
+) (ctxRes context.Context, isSuccessful bool, response zanzibar.RWTStruct, headers map[string]string, e error) {
```

Comment:
First param should always be named `ctx` for consistency. You can name the local member as `ctxRes`

Reply:
ctx is function argument, so the same name can not be used in response arguments